src/Device/deviceLister.ts

```diff
@@ -30,6 +31,7 @@ nrfDeviceLib.setTimeoutConfig(deviceLibContext, { enumerateMs: 3 * 60 * 1000 });
 export const getDeviceLibContext = () => deviceLibContext;
 
 export const logNrfdlLogs = (evt: LogEvent) => {
+    if (remote.app.isPackaged) return;
```

Comment:
Nice, I did not know about `isPackaged`. Maybe we can use this also in other places instead of NODE_ENV.